rero_ils/modules/notifications/api.py

```diff
@@ -69,6 +69,13 @@ class Notification(IlsRecord):
     fetcher = notification_id_fetcher
     provider = NotificationProvider
     model_cls = NotificationMetadata
+    # TODO: understand why we can not test the loan
+    # pids_exist_test = {
+    #     'required': {
+    #         'loan': 'loan',
+    #     },
+    #     'raise_on_erro': True
+    # }
```

Comment:
For now we generate notifications for loans only.

Reply:
ok, deleted